en/faq.md

```diff
@@ -201,6 +201,9 @@ be searched using the query api, but can be used by [visiting](visiting.html).
 #### Visiting does not dump all documents, and/or hangs.
 There are multiple things that can cause this, see [visiting troubleshooting](visiting.html#troubleshooting).
 
+#### How to find number of documents in the index?
+Use metrics or `vespa visit` - see [examples](/en/operations/batch-delete.html#example).
```

Comment:
This is rather expensive compared to running a query for the document types no?

Reply:
true - I'll add a query example, too